public/plugin.tsx

```diff
@@ -157,8 +204,21 @@ export class SecurityPlugin implements Plugin<SecurityPluginSetup, SecurityPlugi
     if (config.multitenancy.enabled) {
       addTenantToShareURL(core);
     }
+
+    if (config.multitenancy.enable_aggregation_view) {
+      const columns = deps.savedObjectsManagement.columns.getAll();
+    }
+
     return {};
   }
 
   public stop() {}
 }
+
+function isPrivateTenant(selectedTenant: string) {
+  return selectedTenant.startsWith('__user__');
```

Comment:
if `selectedTenant` is null, this check errors out, to avoid it we can default the function parameter to empty string like this: `selectedTenant: string = ""`

Reply:
I've changed it to handle the null case.